docs/multiple-base-images.md

```diff
@@ -55,9 +60,10 @@ sematic_pipeline(
 )
 ```
 
-To specify different base images for different steps remove the `base` argument and instead pass a mapping to the `bases` argument:
+To specify different base images for different steps, remove the `base` argument and instead
+pass a mapping to the `bases` argument:
 
-```starlark
+```
```

Comment:
Why remove `starlark`? That's the [language bazel files are written in](https://bazel.build/rules/language)

Reply:
+1